api-playground/migrating-from-mdx.mdx

```diff
@@ -2,7 +2,6 @@
 title: "Migrating MDX API pages to OpenAPI navigation"
 sidebarTitle: "Migrating from MDX"
 description: "Update from individual MDX endpoint pages to automated OpenAPI generation with flexible navigation"
-icon: "arrow-big-right-dash"
```

Comment:
I think we should stick with either having icons for all pages or not have icons. Atm it's inconsistent between the documentation tab, API Reference, and Guides. I would keep the ones for Guides